xbmc/pvr/windows/GUIWindowPVRChannels.cpp

```diff
@@ -150,7 +120,6 @@ bool CGUIWindowPVRChannels::OnAction(const CAction &action)
 {
   switch (action.GetID())
   {
-    case REMOTE_0:
```

Comment:
What do you expect if you press '0' when in channels window? '1-9' is fine to init numeric input dialog imo, but zero?